Recrutify/Recrutify.Host/Infrastructure/Authorization/RolesPolicyHandler.cs

```diff
@@ -28,8 +28,10 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
                 .Select(c => JsonConvert.DeserializeObject<ProjectRoles>(c.Value))
                 .ToDictionary(c => c.ProjectId, c => c.Roles);
 
-            if (projectRoles.TryGetValue(projectId, out var globalRolesValue)
+            if ((projectRoles.TryGetValue(projectId, out var globalRolesValue)
                     && globalRolesValue.Any(r => requirement.Roles.Contains(r)))
+                    || (projectRoles.TryGetValue(DataAccess.Constants.GlobalProject.GlobalProjectId, out var role)
+                    && role.Contains(Constants.Roles.Admin)))
```

Comment:
Rename 
role - claimsGlobalRolesValue
Constants.Roles.Admin - change to nameof(Role.Admin)
and delete Admin from constants